src/test/test-connman.c

```diff
@@ -0,0 +1,84 @@
+/*
+ * This file is part of the Soletta Project
+ *
+ * Copyright (C) 2015 Intel Corporation. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <stdbool.h>
+#include <inttypes.h>
+#include <stdio.h>
+#include <errno.h>
+
+#include "soletta.h"
+#include "sol-connman.h"
+
+#include "sol-util.h"
+#include "sol-log.h"
+
+#include "test.h"
+
+void
+manager_cb(void *data)
+{
+    enum sol_connman_state g_state;
+    bool offline;
+
+    g_state = sol_connman_get_state();
+    printf("system state = %d\n", g_state);
+
+    offline = sol_connman_get_offline();
+    printf("system offline = %d\n", offline);
+}
```

Comment:
If connect a service, we need a agent to response some info from connman.
I will do it after implementing the agent APIs, it will be implemented in next PR.